### PR TITLE
18 update home view

### LINKED ISSUE #1: Remove machine
Implement a "Remove" button to remove a machine from the list of connected machines.

Maybe copy how the scheduledbatches page was made and its remove button.

### LINKED ISSUE #18: Update home view
Convert to functional component.

The view with the machine list is missing information.
The list should likely be converted to a table like that on the scheduled page.
It should also contain ALL the data. Currently, we can only get the IP and ID of the selected machine and it would be nice to have data such as auto-brewing to disable buttons in the control view

### PR DESCRIPTION
**Pull request by Kaszz**
Closes #18 #1 

Converted the MachineList.js to a functional component.
The object CurrentMachine that is passed between all components now holds all the information meaning addition of name and auto-brewing. The list was also converted to a table to hold more data in a uniformly looking way. 
Additionally, removing machine functionality was added as it would be more work to remove it from the table than just copying the working table and changing the endpoint.

Image of how it looks when a machine is currently auto-brewing.

### CODE REVIEW

**Review by Teyson — APPROVED** [review]

**Comment by dottore1** [comment]
Skal connect knappen være så stor? :D

File: src/components/MachineList.js [hunk 1]
```diff
@@ -1,182 +1,267 @@
...
+								<Icon icon="bi:slash-circle-fill" color="#b8352c" width="20" pointerEvents="none" style={{transform: "translateX(0px) translateY(2px)"}}/>}
+							</td>
+																															
+							<td style={{textAlign: "right", width: "10px"}} onClick={removeConnection}><Deletebutton> <Icon icon="akar-icons:cross" color="#fff" width="20" pointerEvents="none" /></Deletebutton></td>
+						</tr>
+					))}
+				</Styledbody>
+			</Styledtable>
+
+			<Link to="/control">
```
Comment (dottore1) [inline src/components/MachineList.js:133]: Skal knappen være så stor? :D

**Review by LeaChr96 — APPROVED** [review]

File: src/components/MachineList.js (hunk 1 above)
Comment (Kaszz) [reply src/components/MachineList.js:133]: Ved ikke hvordan den ellers skal stå, det ser underligt ud hvis den er smallere end hele tabellen.

Comment (Kaszz) [reply src/components/MachineList.js:133]: Men der ligger stadig et issue omkring at beautify.

**Review by Kechr — APPROVED** [review]

**Review by dottore1 — APPROVED** [review]

**Pull request merged**